src/CatenaLogic.Analyzers/Analyzers/NamespacesAnalyzer.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CatenaLogic.Analyzers
+{
+    using System.Collections.Immutable;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal class NamespacesAnalyzer : DiagnosticAnalyzerBase
+    {
+        /// <inheritdoc/>
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } =
+            ImmutableArray.Create(Descriptors.CL0003_DontUseExtensionsNamespace, Descriptors.CL0004_DontUseInterfacesNamespace, Descriptors.CL0005_DontUseHelpersNamespace);
```

Comment:
please add each analyzer on a new line so it's easy to see which ones we support